plugin/trino-http-event-listener/src/main/java/io/trino/plugin/httpquery/HttpEventListener.java

```diff
@@ -82,27 +82,27 @@ public HttpEventListener(HttpEventListenerConfig config, @ForHttpEventListener H
     public void queryCreated(QueryCreatedEvent queryCreatedEvent)
     {
         if (config.getLogCreated()) {
-            sendLog(queryCreatedEvent);
+            sendLog(queryCreatedEvent, queryCreatedEvent.getMetadata().getQueryId());
```

Comment:
Commit messge `Add more logging the event listener` -> `Add more logging in the HTTP event listener`